janus_server/src/aggregator.rs

```diff
@@ -2132,7 +2132,7 @@ fn aggregator_filter<C: Clock>(
             CONTENT_TYPE.as_str(),
             AggregateShareReq::MEDIA_TYPE,
         ))
-        .and(with_cloned_value(aggregator))
+        .and(with_cloned_value(aggregator.clone()))
```

Comment:
nit: `Arc::clone(&aggregator)` (& please fixup the other instances in this function)